### PR TITLE
Updated SyncPrivacyProfileProperties param to new

### PR DESCRIPTION
**Pull request by PramodBalusu**

### CODE REVIEW

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md [hunk 1]
```diff
@@ -1507,11 +1507,12 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
-### -SyncPrivacyProfileProperties
+### -ReSyncTenantPrivacyProfile 
+Parameter 'SyncPrivacyProfileProperties' is obsolete and renamed to ReSyncTenantPrivacyProfile
```
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md:1511]:
```suggestion
The 'SyncPrivacyProfileProperties' parameter is obsolete and renamed ReSyncTenantPrivacyProfile.
```

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md [hunk 2]
```diff
@@ -1507,11 +1507,12 @@ Accept pipeline input: False
...
 
 This parameter enables the synchronization of privacy profile properties.
 
-SyncPrivacyProfileProperties sets whether or not the synced tenant properties will be updated on the next request. The request will cause Azure Active Directory to grab the tenant's current display name (TenantDisplayName) and privacy profile URL (PrivacyProfileUrl) . 
+ReSyncTenantPrivacyProfile sets whether or not the synced tenant properties will be updated on the next request. The request will cause Azure Active Directory to grab the tenant's current display name (TenantDisplayName) and privacy profile URL (PrivacyProfileUrl) .
```
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md:1515]:
```suggestion
ReSyncTenantPrivacyProfile sets whether or not the synced tenant properties will be updated on the next request. The request will cause Azure Active Directory to grab the tenant's current display name (TenantDisplayName) and privacy profile URL (PrivacyProfileUrl). 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @PramodBalusu, kindly commit the suggested changes. Many thanks.

**Review by get-itips — APPROVED** [review]

**Comment by get-itips** [comment]
I am ok after @JohanFreelancer9 suggestions applied

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md [hunk 3]
```diff
@@ -83,7 +83,7 @@ Set-SPOTenant [-ApplyAppEnforcedRestrictionsToAdHocRecipients <Boolean>]
  [-SharingDomainRestrictionMode <SharingDomainRestrictionModes>]
  [-ShowPeoplePickerSuggestionsForGuestUsers <Boolean>]
  [-SpecialCharactersStateInFileFolderNames <SpecialCharactersState>]
- [-SyncPrivacyProfileProperties <Boolean>]
+ [-ReSyncTenantPrivacyProfile <Boolean>]
```
Comment (dariomws) [inline sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md:86]: Not boolean I think:

image

**Review by dariomws — CHANGES_REQUESTED** [review]
Maybe no value is needed:

image

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@PramodBalusu Please review the feedback and update if you agree. Thanks

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md (hunk 3 above)
Comment (PramodBalusu) [reply sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md:86]: Just tested it as well. It's not a Boolean and don't need any additional parameters.

**Comment by yogkumgit** [comment]
@PramodBalusu Gentle reminder.

**Comment by yogkumgit** [comment]
@dariomws

**Review by dariomws — APPROVED** [review]
I approve but I think we must add a type. Not sure which one, maybe String or Switch?

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md (hunk 3 above)
Comment (PramodBalusu) [reply sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md]: Updated it but I'm usure on the Type too.

**Comment by yogkumgit** [comment]
@PramodBalusu

**Comment by PramodBalusu** [comment]
> @PramodBalusu

**Comment by PramodBalusu** [comment]
> @yogkumgit I updated the commit to remove Boolean, anything left to push this to live ? We have a customer waiting to see this change

**Comment by yogkumgit** [comment]
@tseward

**Review by tseward — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**